### PR TITLE
Update formatting-a-csv.md

### PR DESCRIPTION
**Pull request by misselliev**
Updated punctuation.

**Updated punctuation**:
**Improving reading flow**:
**Link to original source**:

Closes #

### CODE REVIEW

File: content/docs/ui/managing-contacts/formatting-a-csv.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ If your contacts list has non-English characters, please make sure that you're u
 SendGrid removes duplicate and invalid email addresses (including email addresses with special characters) from your list during upload, so the number of contacts can potentially change between your CSV and your Marketing Campaigns list.
 
 ###  Numbers and Text
-Make sure that you don’t have text fields that look like numbers. Text that look like numbers are those that contain punctuation characters like a dash, underscore, parenthesis, or multiple dots.
+Make sure that you don’t have text fields that look like numbers. Text that look like numbers are those that contain punctuation characters like a dash, underscore, parenthesis or multiple dots.
```
Comment (Whatthefoxsays) [inline content/docs/ui/managing-contacts/formatting-a-csv.md]: We ❤️the Oxford comma here at SendGrid! Can you please add it back in? Thanks!

**Comment by misselliev** [comment]
Sorry, added the comma!

**Comment by thinkingserious** [comment]
Hello @misselliev,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @misselliev,<br /><br/>Thanks again for the PR!<br /><br />                                         You have earned 6 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold or to simply continue the celebration.<br /><br />                                         Also, <strong>please be sure you have officially registered with us <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a> by November 1, 2018</strong> to qualify.<br /><br />                                         If you have any questions you can email us at <a href="mailto:dx+hacktoberfest2018@sendgrid.com">dx+hacktoberfest2018@sendgrid.com</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid

**Pull request merged**